python/py-tensorflow-tensorboard/Portfile

```diff
@@ -17,19 +17,19 @@ long_description    TensorBoard is a suite of web applications for inspecting \
 homepage            https://github.com/tensorflow/tensorboard
 
 if {${python.version} < 30} {
-    master_sites        https://pypi.python.org/packages/d5/fb/80df4eb3234c41beeeb3122b1966effbf08608fb80d2fa74bb72f8da9cb3
-    distname            tensorflow_tensorboard-${version}-py2-none-any
-
-    checksums           rmd160  52c4d0d3bf8856e2ebc87ae9785319e4416ccdce \
-                        sha256  2b8741aed01f35c95b781d25ebf1a84a45c2980746a616a36ade7ed8078d88ee
+    master_sites        https://files.pythonhosted.org/packages/76/f9/e62022d00940e4df9a629d6bfe42eb28907bb35808db62bb9e8b69ea5ef3
```

Comment:
You may want to use the hash-free URL: `https://files.pythonhosted.org/packages/cp${python.version}/t/tensorflow`

Reply:
Thanks for the hint, that is a nice simplification.